modules/tensor_mechanics/doc/content/source/materials/ADLAROMANCEStressUpdateBase.md

```diff
@@ -13,6 +12,10 @@ function of temperature, defect concentrations, the von Mises trial stress, and
 factor. In addition, lower-length scale information, here mobile defect dislocations and immobile
 defect dislocations, are evolved as determined by the ROM.
 
+`ADLAROMANCEStressUpdateBase` provides the necessary math and implementation for ROMs provided in
+the correct `LAROMANCE` model format, which essentially includes the necessary material specific
+data. An example of how to implement the necessary data is provided in `SS316HLAROMANCEStressUpdateTest`.
```

Comment:
Do the test objects have documentation pages to which one can link in this documentation page?

Reply:
I wasn't sure if you could make doco for test objects, or if that was a smart idea even if you could?

Reply:
I'm not sure either, but I am sure that @aeslaughter has the answer: docs for tests classes, yes or no?

Reply:
Yeah, this was a bit tough. I tried to emulate the documentation for RadialReturn. I'll add some words about SS316HLAROMANCEStressUpdateTest